src/init/features/dataconnect/sdk.ts

```diff
@@ -26,9 +26,14 @@ import {
 import { DataConnectEmulator } from "../../../emulator/dataconnectEmulator";
 import { FirebaseError } from "../../../error";
 import { camelCase, snakeCase, upperFirst } from "lodash";
-import { logSuccess, logBullet, promptForDirectory } from "../../../utils";
+import { logSuccess, logBullet, promptForDirectory, envOverride } from "../../../utils";
 import { getGlobalDefaultAccount } from "../../../auth";
 
+// connectorEnvVar is used by Firebase Console to specify which connector to setup.
+// It should be in the form <connectorId>.
```

Comment:
This comment is misleading. For example, your PR description contained one with several slashes and it worked